src/client/content_cao.cpp

```diff
@@ -1841,8 +1841,6 @@ void GenericCAO::processMessage(const std::string &data)
 		(void)type;
 
 		addAttachmentChild(child_id);
-	} else if (cmd == AO_CMD_OBSOLETE1) {
```

Comment:
@rubenwardy I suppose I should keep this too or is it fine?

Reply:
Should be kept.